### ISSUE TITLE
Can't upload on web, "MultipartFile is only supported where dart:io is available."

### ISSUE THREAD

**Issue by esDotDev** [djade007/cloudinary_public#7]
On Flutter Web, I'm calling `cloudStorage.multiUpload` and getting:
```
Error: Unsupported operation: MultipartFile is only supported where dart:io is available.
    at Object.throw_ [as throw] (http://localhost:52896/dart_sdk.js:5342:11)
```

Is this expected? Seems like this might be an issue in Dio, so not sure exactly where to log it.

**Comment by esDotDev** [comment]
Created an issue here: https://github.com/flutterchina/dio/issues/1067

**Comment by djade007** [comment]
@esDotDev I'm looking into this now. There was an initial plan to switch to https://pub.dev/packages/http instead of Dio

**Comment by djade007** [comment]
Update:

I was able to get it to work on web this way:

`
    try {
      final res = await cloudinary.uploadFile(
        CloudinaryFile.fromByteData(
          ByteData.sublistView(
            await _pickedFile.readAsBytes(),
          ),
          identifier: _pickedFile.path.split('.').last,
        ),
      );
      print(res);
    } catch (e, s) {
      print(e);
      print(s);
    }
`

**Comment by djade007** [comment]
@esDotDev https://github.com/djade007/cloudinary_public/blob/master/example/lib/src/image_picker_example.dart

**Comment by esDotDev** [comment]
Amazing, thanks!

**Comment by djade007** [comment]
@esDotDev I have published a new version `0.7.0`

- It now uses http instead of Dio
- Your initial code should also work on web, you don't have to use the code example I provided above.
